docs/Encoder.md

```diff
@@ -1,57 +1,57 @@
+|--------------------------------------|--------|---------------|
+| Supply voltage Vcc                   |        | 7 V           |
+| Input voltage (A, A̅, A, B̅, Z or Z̅)   | -11 V  | 14 V          |
+| Differential input voltage           | -14 V  | 14 V          |
+| Output voltage (OUT A, OUT B, OUT Z) | -0.5 V | (Vcc + 0.5) V |
 
-### 4. Level Shifter
-After encoder differential signals are converted to single-ended, these signals are then level-shifted/translated to a lower voltage, allowing compatibility with PicoZed. The level shifter [SN74LVC8T245](http://www.ti.com/lit/ds/symlink/sn74lvc8t245.pdf) uses two separate configurable power-supply rails and allows for bidirectional operation. Signals on each side (input or output) are referenced to their respective voltage rail. In AMDC Rev D, the input voltage rail is at 5V and the output voltage rail is at 1.8 V. The maximum output current can be up to 32 mA when
-device is powered by 5 V. The outputs of this chip (6 signals in total if 2 encoders are connected) are then transmitted to PicoZed.
-The rise and fall times of the output signal are both approximately equal to 1ns. Assuming the system as first order, the bandwidth of the chip would approximately be equal to 0.35/1 ns = 350 MHz.
+Exposure to absolute maximum ratings for an extended period of time may affect device reliability. Therefore, it is recommended to operate under the recommended operating conditions:
 
-## PCB Layout
+| Parameter                             | MIN   | NOM | MAX   |
+|---------------------------------------|-------|-----|-------|
+| Supply voltage Vcc                    | 4.5 V | 5 V | 5.5 V |
+| High-level input voltage (logic HIGH) | 2 V   |     | Vcc   |
+| Low-level input voltage (logic LOW)   | 0 V   |     | 0.8 V |
+| Common-mode input voltage             | -7 V  |     | 7 V   |
 
-All chips and d-sub connectors are located on the top layer. Signal routes from d-sub connectors to differential to single-ended converter and then to level shifter are all located on the top layer as well, while signals from level shifter to PicoZed are located in the intermediate level and each inverting and non-inverting pair of encoder signals are connected between each other by the termination resistor on the bottom layer. All signal traces have 6 mil thickness. Supply and ground traces connected to the chips have thickness of 10 mil.
+When the chip is operated at nominal conditions (Vcc = 5 V, room temperature 25 C), the output signals of the chip are such that the high-level output voltage is 3.8 V or more and low-level output voltage is around 0.2 V. Typical current the chip consumes is about 10 mA. This translates to about 50 mW of power consumption with 5 V supply. The rise and fall times of the output signal are both approximately equal to 4 ns. Assuming the system as first order, the bandwidth of the chip is approximately equal to 0.35/4 ns = 87.5 MHz.

```

Comment:
> The rise and fall times of the output signal are both approximately equal to 4 ns. Assuming the system as first order, the bandwidth of the chip is approximately equal to 0.35/4 ns = 87.5 MHz.

By bandwidth, are you trying to indicate the fastest encoder signal that can be brought into the AMDC? If so, I am not sure that this is the best way to indicate that. Usually digital circuits specify a maximum frequency they can reliably operated at.

Reply:
```suggestion
When the chip is operated at nominal conditions (Vcc = 5 V, room temperature 25 C), the output signals of the chip are such that the high-level output voltage is 3.8 V or more and low-level output voltage is around 0.2 V. Typical current the chip consumes is about 10 mA. This translates to about 50 mW of power consumption with 5 V supply. The rise and fall times of the output signal are both approximately equal to 4 ns. Assuming the system as first order, the maximum frequency the chip can reliably operate at is approximately 0.35/4 ns = 87.5 MHz. 

```